vncviewer/DesktopWindow.cxx

```diff
@@ -174,7 +174,7 @@ DesktopWindow::DesktopWindow(int w, int h, const char *name,
 #ifdef __APPLE__
   // On OS X we can do the maximize thing properly before the
   // window is showned. Other platforms handled further down...
-  if (maximize) {
+  if (false) {
```

Comment:
Oops? :)

Reply:
Oops indeed, hardcoded some things to build with fltk-1.4